manifests/mod/auth_cas.pp

```diff
@@ -87,28 +87,28 @@
 class apache::mod::auth_cas (
   String $cas_login_url,
   String $cas_validate_url,
-  String $cas_cookie_path                     = $apache::params::cas_cookie_path,
-  Stdlib::Filemode $cas_cookie_path_mode      = '0750',
-  Integer $cas_version                        = 2,
-  String $cas_debug                           = 'Off',
-  Optional[String] $cas_validate_server       = undef,
-  Optional[String] $cas_validate_depth        = undef,
-  Optional[String] $cas_certificate_path      = undef,
-  Optional[String] $cas_proxy_validate_url    = undef,
-  Optional[String] $cas_root_proxied_as       = undef,
-  Optional[String] $cas_cookie_entropy        = undef,
-  Optional[String] $cas_timeout               = undef,
-  Optional[String] $cas_idle_timeout          = undef,
-  Optional[String] $cas_cache_clean_interval  = undef,
-  Optional[String] $cas_cookie_domain         = undef,
-  Optional[String] $cas_cookie_http_only      = undef,
-  Optional[String] $cas_authoritative         = undef,
-  Optional[String] $cas_validate_saml         = undef,
-  Optional[String] $cas_sso_enabled           = undef,
-  Optional[String] $cas_attribute_prefix      = undef,
-  Optional[String] $cas_attribute_delimiter   = undef,
-  Optional[String] $cas_scrub_request_headers = undef,
-  Boolean $suppress_warning                   = false,
+  String $cas_cookie_path                                = $apache::params::cas_cookie_path,
+  Stdlib::Filemode $cas_cookie_path_mode                 = '0750',
+  Integer $cas_version                                   = 2,
+  String $cas_debug                                      = 'Off',
+  Optional[String] $cas_validate_server                  = undef,
+  Optional[String] $cas_validate_depth                   = undef,
+  Optional[String] $cas_certificate_path                 = undef,
+  Optional[String] $cas_proxy_validate_url               = undef,
+  Optional[String] $cas_root_proxied_as                  = undef,
+  Optional[String] $cas_cookie_entropy                   = undef,
+  Optional[Variant[Integer[0],String]] $cas_timeout      = undef,
```

Comment:
could you add a unit test for the three params that set them to an integer? Is there an upper limit within apache that we could enforce with the type?